centinel/data/country.txt

```diff
@@ -0,0 +1,2 @@
+"yahoo.fr/this,and,that",xx,yy
```

Comment:
Comma's are allowed in URL paths, (they are reserved characters in domain names).
Citizen Lab's URLs are sometimes enclosed in quotes because they contain comma's in their paths.